kubernetes/config-maps/concent-api/local_settings.py.j2

```diff
@@ -30,6 +30,7 @@ ADMINS = [
 EMAIL_HOST          = "{{ email.host }}"
 EMAIL_PORT          = {{ email.port }}
 EMAIL_HOST_USER     = "{{ email.user }}"
+EMAIL_SUBJECT_PREFIX = "[{{ gke.cluster }}-concent-api] "
```

Comment:
This is not what I requested in https://github.com/golemfactory/concent-deployment/pull/41#discussion_r163215396

Reply:
Ok, I see. Replace "-"=> ":"